### ISSUE TITLE
Add a11y Lint Checks

### ISSUE THREAD

**Issue by AmandaRiu** [woocommerce/woocommerce-android#342]
Add Lint checks to enforce some accessibility standards. We may be able to reuse the [WordPress-Lint-Android project](https://github.com/wordpress-mobile/WordPress-Lint-Android/) and just copy the lint.xml file over.

**Comment by JorgeMucientes** [comment]
Closing this as part of a maintenance pass on the GitHub issue backlog. This issue matched the auto-close criteria for this pass: it has not been updated since 2021-08-09, is unassigned, is not labeled as a bug or high/medium priority, and does not appear to involve POS, payments, login, data loss, crash, security, vulnerability, auth, or fraud.

If this is still relevant, please comment with current details and we can reopen it or move it into Linear for triage.
